src/handlers.py

```diff
@@ -150,6 +169,35 @@ async def catch_receipt(message: types.Message):
         await message.answer("Ой, как же больно! Что-то сломалось внутри меня.")
 
 
-async def get_receipts(message: types.Message):
+async def get_email_for_sending(message: types.Message):
+    await SendingMail.email.set()
+    await message.answer("Пожалуйста, напишите e-mail для отправки файла: ")
+
+
+async def get_receipts(message: types.Message, state: FSMContext):
```

Comment:
Let's divide it into two independent handlers:
First handler allows one to add subscriber email. Should be available for Tg admin only.
Second handler allows one to initiate sending of Excel. Should be available for Tg admin only.